Source/Core/DiscIO/VolumeWiiCrypted.cpp

```diff
@@ -36,6 +36,12 @@ CVolumeWiiCrypted::CVolumeWiiCrypted(std::unique_ptr<IBlobReader> reader)
 {
   _assert_(m_pReader);
 
+  if (m_pReader->ReadSwapped<u32>(0x60) != u32(0))
```

Comment:
u makes sure that it's unsigned, but it'll still just be an `unsigned int` and not a `u32`. I don't see why hexadecimal would help.